pkg/i2gw/ingress2gateway.go

```diff
@@ -64,15 +64,15 @@ func ToGatewayAPIResources(ctx context.Context, namespace string, inputFile stri
 			return nil, nil, fmt.Errorf("failed to read ingresses from file: %w", err)
 		}
 		resources.Ingresses = ingresses.Items
-		if err = readProviderResourcesFromFile(ctx, providerByName, &resources, inputFile); err != nil {
+		if err = readProviderResourcesFromFile(ctx, providerByName, inputFile); err != nil {
```

Comment:
I think we need to change `Read` to `Fetch` also for this one and the `FromCluster` one

Reply:
Done.